src/control/ToolEnums.cpp

```diff
@@ -172,8 +172,8 @@ auto toolTypeFromString(const string& type) -> ToolType {
     if (type == "drawRect") {
         return TOOL_DRAW_RECT;
     }
-    if (type == "drawCircle") {
-        return TOOL_DRAW_CIRCLE;
+    if (type == "drawEllipse") {
```

Comment:
```suggestion
    // recognize previous spelling of Highlighter, V1.1.0+dev (Jan 2021) and earlier
    if (type == "drawEllipse" || type == "drawCircle") {
```

Reply:
Don't you mean
`// recognize previous spelling of Ellipse, V1.1.0+dev (Jan 2021) and earlier`
?

Reply:
Yes, sorry for the typo. You can amend it to the current one.